### ISSUE TITLE
Uninitialized constant Users error for expired links

### ISSUE THREAD

**Issue by adrienpoly** [abevoelker/devise-passwordless#3]
Hello 

Thanks for this great Gem.

For an expired link I am seeing a strange error that I have hard time understanding how to fix it.

The error happens in the call to warden

```ruby
self.resource = warden.authenticate!(auth_options)
```

![Capture d’écran 2021-04-26 à 7 27 00 AM](https://user-images.githubusercontent.com/7847244/116032925-cd844100-a660-11eb-9436-19d067db888a.jpg)

I tried this but still getting the same error

```ruby
  def show
    self.resource = warden.authenticate!(auth_options)
    set_flash_message!(:notice, :signed_in)
    sign_in(resource_name, resource)
    yield resource if block_given?
    redirect_to after_sign_in_path_for(resource)
  rescue
    flash[:notice] = t("devise.failure.user.magic_link_invalid")
    redirect_to root_path
  end
```

**Comment by abevoelker** [comment]
Oh that’s not good. Can you share your routes from config/routes.rb?

**Comment by adrienpoly** [comment]
sure the project is open source here https://github.com/hostolab/covidliste/blob/master/config/routes.rb

Lots of the issues and PR etc are in french but the code is all in English 😄

**Comment by adrienpoly** [comment]
This is awesome, thanks for your quick answer 🚀

**Comment by adrienpoly** [comment]
I can confirm that it fixed the issue on our side thanks

**Comment by abevoelker** [comment]
Great, thanks for letting me know! I've released 0.6 version of the gem with a fixed generator.
